README.md

```diff
@@ -180,6 +181,34 @@ jobs:
           config: .github/buildkitd.toml
 ```
 
+### Standalone mode
+
+If you don't have the Docker engine and CLI installed on your GitHub Runner,
+you can use buildx in standalone mode. This can be useful if you want to use
```

Comment:
You can add "mode. In this case buildx binary is invoked directly, instead of calling it as a docker plugin."